react/QuotaAlert/index.jsx

```diff
@@ -46,6 +37,19 @@ const QuotaAlert = ({ t, onClose, client }) => {
       dismissAction={onClose}
     />
   )
-}
+})
+
+const QuotaAlert = withClient(({ client, onClose }) => {
+  /**
+   * We do the request only on the web since Apple
```

Comment:
Which request ? AMHA the comment here is not very clear.

Reply:
Yep, now that I moved the call it's not clear. fixed